README.md

```diff
@@ -7,6 +7,13 @@ Optune servo driver for Prometheus
 2. The default Prometheus endpoint is `http://prometheus:9090/metrics`; you must
    set the environment variable `PROMETHEUS_ENDPOINT` in the servo's container
    if you want to use a different endpoint.
+3. Alternatively, Prometheus endpoint may be specified in `prometheus_endpoint` key
+   of driver configuration. If this key exists and is not empty, value specified will
+   override `PROMETHEUS_ENDPOINT`.
```

Comment:
Personally I would prefer to have the environment variable to take precedence.

Reply:
Environment variable is not applicable when multiple drivers are used under MAGG, as the value would apply to all drivers, defying the purpose. Defining the precedence as 'env var unless you do multiple drivers, then config value' leads to inconsistent behaviour that, to my mind, is worse.